src/designer/elsa-workflows-studio/src/modules/credential-manager/models/secrets/oauth2.secret.ts

```diff
@@ -74,17 +74,41 @@ export const OAuth2: Secret = {
       type: 'System.String',
       uiHint: 'single-line',
     },
+    {
+      disableWorkflowProviderSelection: false,
+      isBrowsable: true,
+      isReadOnly: false,
+      label: 'Credentials location',
+      name: 'CredentialsLocation',
```

Comment:
Note: in case you'd want to use the standard term, it's officially called "token endpoint client authentication method".